src/QsCompiler/Tests.Compiler/TestCases/PopulateCallGraph.qs

```diff
@@ -134,6 +134,81 @@ namespace Microsoft.Quantum.Testing.PopulateCallGraph {
 
 // =================================
 
+// Multiple Entry Points
+namespace Microsoft.Quantum.Testing.PopulateCallGraph {
+
+    @EntryPoint()
+    operation Main1() : Unit {
+        Foo();
+    }
+
+    @EntryPoint()
+    operation Main2() : Unit {
+        Bar();
+    }
+
+    operation Foo() : Unit { }
+
+    operation Bar() : Unit { }
+}
+
+// =================================
+
+// Unit Test as Starting Point
+namespace Microsoft.Quantum.Testing.PopulateCallGraph {
+
+    open Microsoft.Quantum.Diagnostics;
+
+    @Test("QuantumSimulator")
+    operation Test() : Unit {
+        Foo();
+    }
+
+    operation Foo() : Unit { }
+
+    operation Bar() : Unit { }
+}
+
+// =================================
+
+// Entry Points and Unit Tests
+namespace Microsoft.Quantum.Testing.PopulateCallGraph {
+
+    open Microsoft.Quantum.Diagnostics;
+
+    @EntryPoint()
+    operation Main1() : Unit {
+        Foo();
+    }
+
+    @EntryPoint()
+    operation Main2() : Unit {
+        Bar();
+    }
+
+    @Test("QuantumSimulator")
+    operation Test1() : Unit {
+        Zip();
+    }
+
+    @Test("QuantumSimulator")
+    operation Test2() : Unit {
+        Zap();
+    }
+
+    operation Foo() : Unit { }
+
+    operation Bar() : Unit { }
+
+    operation Zip() : Unit { }
+
+    operation Zap() : Unit { }
+
+    operation Unused() : Unit { }
```

Comment:
(Minor)
Would be nice to see a code comment telling what this `Unused()` is for.

Reply:
Unused is to test that unused callables are either included in the call graph or not, depending on the situation and unit test.

Reply:
I could add a comment to the corresponding test logic, but then commenting a line line `AssertNotInConcreteGraph graph Unused` seems a bit unnecessary to me. It's hard to be more descriptive than that 😊.